common/persistence/persistence-tests/metadataPersistenceV2Test.go

```diff
@@ -611,6 +625,7 @@ func (m *MetadataPersistenceSuiteV2) TestConcurrentUpdateDomain() {
 					VisibilityArchivalStatus: resp2.Config.VisibilityArchivalStatus,
 					VisibilityArchivalURI:    resp2.Config.VisibilityArchivalURI,
 					BadBinaries:              testBinaries,
+					IsolationGroups:          isolationGroups,
```

Comment:
Could you use a different IsolationGroupConfiguration and check the value is equal to the new value after UpdateDomain?